src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -2242,6 +2242,13 @@ public Iterable<Translation> getArticleTranslations(Long articleId) {
                 handleList(Translation.class, "translations"));
     }
 
+    public Translation getArticleTranslation(long articleId, String locale) {
```

Comment:
Shouldn't we use the same verb as the Zendesk API: https://developer.zendesk.com/api-reference/help_center/help-center-api/translations/#show-translation:

```suggestion
    public Translation showArticleTranslation(long articleId, String locale) {
```